lib/iceserversource/nts.js

```diff
@@ -206,8 +216,11 @@ function poll(client) {
     if (!config) {
       throw alreadyStopped;
     }
-    return parseECSConfig(client, config);
+    const iceServersAndTTL = parseECSConfig(client, config);
+    client._status = 'success';
+    return iceServersAndTTL;
   }).catch(error => {
+    client._status = 'failure';
```

Comment:
@markandrus If the default value is "failure" is this line really necessary?

Reply:
This will poll multiple times, possibly succeeding or failing. Granted we only advertise/use the result once at connect-time, but still, I think we should set it here. I'm also making the default value `null`, so we'll need to keep this.